src/Widgets/VpnInterface.vala

```diff
@@ -134,6 +135,8 @@ public class Network.VpnInterface : Network.WidgetNMInterface {
         check_vpn_availability ();
         if (active_vpn_item != null) {
             vpn_item.active = true;
+        } else {
+            vpn_item.active = false;
         }
```

Comment:
My knowledge of Vala is limited and my knowledge of the accepted coding practices in elementary OS -- even more so, so this suggestion might not be a good one.

But I'm experienced developer working mostly in Java lately, and I think the following is easier to read:

```suggestion
        vpn_item.active = active_vpn_item != null;
```

Reply:
Yes. It's much more elegant that way.